### ISSUE TITLE
lscpu doesn't work

### ISSUE THREAD

**Issue by petkar** [SDRausty/TermuxArch#240]
`lscpu` command doesn't work. Adding the string `-b /sys/devices/system/cpu/:/sys/devices/system/cpu/` to proot or just binding `sys` should fix it. Any particular reason it wasn't bound by default?

**Comment by SDRausty** [comment]
This commit https://github.com/TermuxArch/TermuxArch/commit/5a0763949dc99d4f39417b0075cc0c0fca5066d7 fixes it.  `startarch s user lscpu` can be used to test it.  Thank you for sharing @petkar.

**Comment by petkar** [comment]
I'm thankful, but doing `setupTermuxArch.sh r` breaks my display by locking x. This happened twice until now and I wasn't able to determine the cause.
[This](https://github.com/SDRausty/TermuxArch/issues/228#issuecomment-660830867) should give some insight. I'll be sure to test it on another device.

**Comment by SDRausty** [comment]
We can open the `gens` folder once more for development if we think it will assist;  What is the output of the `startarch` that works best?
```
[08:38TermuxArch]$ grep exec\ proot ~/arch/startarch
exec proot --kill-on-exit --link2symlink -0 -r /data/data/com.termux/files/home/arch -b /apex:/apex -b /dev/ashmem:/dev/ashmem -b /storage/:/storage/ -b /data/data/com.termux/files/home:/data/data/com.termux/files/home -b /sdcard:/sdcard -b /data/data/com.termux/files/usr:/data/data/com.termux/files/usr -b /proc/:/proc/ -b /vendor/:/vendor/ -b /system/:/system/ -b /data/data/com.termux/files/home/arch/tmp:/dev/shm -b /data/data/com.termux/files/home/arch/var/binds/fbindprocstat:/proc/stat -b /sys/:/sys/ -b /dev/:/dev/ -w "$PWD" /usr/bin/env -i HOME=/root TERM="$TERM" ANDROID_DATA=/data  /bin/bash -l ||:
exec proot --kill-on-exit --link2symlink -0 -r /data/data/com.termux/files/home/arch -b /apex:/apex -b /dev/ashmem:/dev/ashmem -b /storage/:/storage/ -b /data/data/com.termux/files/home:/data/data/com.termux/files/home -b /sdcard:/sdcard -b /data/data/com.termux/files/usr:/data/data/com.termux/files/usr -b /proc/:/proc/ -b /vendor/:/vendor/ -b /system/:/system/ -b /data/data/com.termux/files/home/arch/tmp:/dev/shm -b /data/data/com.termux/files/home/arch/var/binds/fbindprocstat:/proc/stat -b /sys/:/sys/ -b /dev/:/dev/ -w "$PWD" /usr/bin/env -i HOME=/root TERM="$TERM" ANDROID_DATA=/data  /bin/bash -lc "$ar2ar" ||:
exec proot --kill-on-exit -0 -r /data/data/com.termux/files/home/arch -b /apex:/apex -b /dev/ashmem:/dev/ashmem -b /storage/:/storage/ -b /data/data/com.termux/files/home:/data/data/com.termux/files/home -b /sdcard:/sdcard -b /data/data/com.termux/files/usr:/data/data/com.termux/files/usr -b /proc/:/proc/ -b /vendor/:/vendor/ -b /system/:/system/ -b /data/data/com.termux/files/home/arch/tmp:/dev/shm -b /data/data/com.termux/files/home/arch/var/binds/fbindprocstat:/proc/stat -b /sys/:/sys/ -b /dev/:/dev/ -w "$PWD" /usr/bin/env -i HOME=/root TERM="$TERM" ANDROID_DATA=/data  /bin/su - "$ar2ar" ||:
exec proot --kill-on-exit --link2symlink -0 -r /data/data/com.termux/files/home/arch -b /apex:/apex -b /dev/ashmem:/dev/ashmem -b /storage/:/storage/ -b /data/data/com.termux/files/home:/data/data/com.termux/files/home -b /sdcard:/sdcard -b /data/data/com.termux/files/usr:/data/data/com.termux/files/usr -b /proc/:/proc/ -b /vendor/:/vendor/ -b /system/:/system/ -b /data/data/com.termux/files/home/arch/tmp:/dev/shm -b /data/data/com.termux/files/home/arch/var/binds/fbindprocstat:/proc/stat -b /sys/:/sys/ -b /dev/:/dev/ -w "$PWD" /usr/bin/env -i HOME=/root TERM="$TERM" ANDROID_DATA=/data  /bin/"$ar2ar" ||:
exec proot --kill-on-exit -0 -r /data/data/com.termux/files/home/arch -b /apex:/apex -b /dev/ashmem:/dev/ashmem -b /storage/:/storage/ -b /data/data/com.termux/files/home:/data/data/com.termux/files/home -b /sdcard:/sdcard -b /data/data/com.termux/files/usr:/data/data/com.termux/files/usr -b /proc/:/proc/ -b /vendor/:/vendor/ -b /system/:/system/ -b /data/data/com.termux/files/home/arch/tmp:/dev/shm -b /data/data/com.termux/files/home/arch/var/binds/fbindprocstat:/proc/stat -b /sys/:/sys/ -b /dev/:/dev/ -w "$PWD" /usr/bin/env -i HOME=/root TERM="$TERM" ANDROID_DATA=/data  /bin/su - "$2" -c "$ar3ar" ||:
```

Is the option:
```
[08:41TermuxArch]$ bash setupTermuxArch.bash h|grep loom
 [b[loom]] Create and run a local copy of TermuxArch in TermuxArchBloom. Useful for running a customized setupTermuxArch.bash locally, for developing and hacking TermuxArch.
```
helping in development of the PRoot init statement?

**Comment by petkar** [comment]
After trying to reproduce [the issue](https://github.com/SDRausty/TermuxArch/issues/228#issuecomment-660830867) on another device, I realized this wasn't an issue with proot statement, but entirely my fault. The error was caused because I had installed zsh, and the `.bashrc` and `.profile` (which happens to be important) wasn't exported. As a result the locales got messed, the bin got unlinked, path and gpgtty removed and effectively broke xauth. The solution was to load `.profile` by adding `[[ -e ~/.profile ]] && emulate sh -c 'source ~/.profile'` in `.zshrc`
